frontend/src/components/ColorPickerToolbar.jsx

```diff
@@ -1,10 +1,15 @@
 import "../css/ColorPickerToolbar.css";
 import { FaExchangeAlt } from "react-icons/fa";
 
-function ColorPickerToolbar({primaryColor, setPrimaryColor, secondaryColor, setSecondaryColor}) {
+function ColorPickerToolbar({primaryColor, setPrimaryColor, secondaryColor, setSecondaryColor, handleColorPickerInput }) {
   const handleColorPickerFirst = (event) => {
+    if(handleColorPickerInput){
+      setPrimaryColor(handleColorPickerInput)
```

Comment:
Are you setting the color to the input element itself here? Not quite sure what the intention is